setup.cfg

```diff
@@ -0,0 +1,7 @@
+[flake8]
+ignore = E501,E402
+
+[tool:pytest]
+testpaths = wagtailautocomplete docs
+DJANGO_SETTINGS_MODULE = wagtailautocomplete.tests.settings
+django_find_project = false
```

Comment:
Is this to basically make it easier to run `pytest` and `flake8` outside of `tox`? I can get behind that.

Reply:
I think it is just an arbitrary choice to centralize the configuration of all tools in one *independent* file, since both `pytest` and `flake8` are looking for `tox.ini`...